formats/shared/definitions/finder.jsonnet

```diff
@@ -190,6 +191,16 @@
             "$ref": "#/definitions/label_value_pair",
           },
         },
+        option_lookup: {
+          description: "A map of keys to values that can be used to associate allowed_values with multiple values",
+          type: "object",
+          items: {
```

Comment:
I'm not sure that items is valid here. I have a feeling the below would work. What you could do is try it by generating the schema locally, then validating your example with something like this:

```
GovukContentSchemaTestHelpers::Validator.new("finder", "schema", your_json)
```

```
        option_lookup: {
          description: "A map of keys to values that can be used to associate allowed_values with multiple values",
          type: "object",
          additionalProperties: true,
          patternProperties: {
            "^[a-z_]+$": {
              type: "array",
              items: {
                type: "string"
              }
            }
          }
```

Reply:
Fixed, thanks.